new release 2314 (#1220)
elasticio/elasticio.github.io: content/_releases/23/14.md

---
title: Product Update - v23.14
description: Details of the product update v23.12 from 23th of March 2023.
layout: rel
version: "23.14"
section: Product Updates
category: product-updates
releaseDate: 2023-04-12
redirect_from:
  - /releases/23.14.html
---

## IMPORTANT: New Releases Procedure

> Kindly note that our release process will be changed. The next and all future releases will be once per month (the last week of each month).

Reviewer: ```suggestion
Kindly note that our release process is changing. In future, we will release updates once per month (in the last week of each month).
```

> Generally, all delivery timelines will not be extended as all feature requests are currently prepaid in one sprint(2 weeks iteration) and delivered in the next (2 weeks iteration). In the new process, the feature request and delivery will be in one sprint (1 month iteration).

Reviewer: ```suggestion
This change increases efficiency for the development teams and should not significantly influence delivery times for feature requests. 
```

> _Note: Additional releases are available in case of any critical hotfixes or urgent feature updates._ 

Reviewer: ```suggestion
_Note: Additional releases are possible in case of any critical changes or urgent feature updates._ 
```

4. Added `values-secondary.yaml` file to `admiral` and `platform` charts for disabling services that do NOT run on secondary clusters
5. Added ability in `deploy.sh` to deploy the platform to secondary cluster (i.e. using `values-secondary.yaml`). The command example:
```
KUBE_CONTEXT=great-moraq-stannis USE_SECONDARY=true ./deploy.sh deploy great-moraq
```
Changes required in configurations in k8s clusters (e.g. stage-clusters unmerged PR):
1. In `haproxy.yaml` add `expose: true` and `loadBalancerIp: "{{LOAD_BALANCER_IP}}"` 
2. In `platoform.yaml`:
  
    2.1. In `global.services.api` add `expose: true` and `loadBalancerIp: "{{LOAD_BALANCER_IP}}"` 
  
    2.2. In `global.services.maester` add `expose: true` and `loadBalancerIp: "{{LOAD_BALANCER_IP}}"` 
  
    2.3. In `global.services.steward` add `expose: true` and `loadBalancerIp: "{{LOAD_BALANCER_IP}}"` 
 
    2.4. In `global.services.pssLoadBalancer` add `expose: true`

3. In `secrets.yaml` change all occurrences of `haproxy-service.platform.svc.cluster.local` to an actual haproxy loadbalancer IP address

{% endif %}

> ### "Logs" page: add step name

Author: This info is related only to separate installations(white-label clients).

> ### "Logs" page: add step name

Author: There is no reason to display this information to all platform users.

> We added the `Step Name` column next to the `Flow Name` on the Logs page

Reviewer: ```suggestion
We added the `Step Name` column next to the `Flow Name` on the Logs page. This makes it easier to isolate relevant logs during trouble shooting.
```
